### ISSUE TITLE
Use the detailed, long description as an additional means to explain the forecast

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#1792]

**Comment by colinmurphy01** [comment]
@partly-igor I think you mentioned this in passing. Are you considering this as part of your designs? I didn't see it so I'm assuming no, but just checking.

**Comment by partly-igor** [comment]
No, just thought about them here and there, but I haven't done anything concrete with them.

**Comment by colinmurphy01** [comment]
Roger, I'm thinking we wait on this till we do a round of testing on our new user experience (at some point in the future).
